package.json

```diff
@@ -40,10 +40,11 @@
     "gulp-mocha": "2.2.0",
     "mocha": "2.3.4",
     "s": "0.1.1",
-    "superagent": "0.15.4"
+    "superagent": "0.15.4",
+    "uws": "^0.3.3"
```

Comment:
I guess fix version is okay but in that case you will get many PRs from me, wanting to up the version when I fix things.